test/puppetlabs/http/client/metrics_test.clj

```diff
@@ -71,6 +71,17 @@
       app
       [jetty9/jetty9-service test-metric-web-service]
       {:webserver {:port 10000}}
+      (with-open [client (Async/createClient (doto (ClientOptions.)
+                                               (.setMetricRegistry (MetricRegistry.))
+                                               (.setUseURLMetrics false)))]
+        (let [request-opts (RequestOptions. hello-url)
+              response (-> client (.get request-opts) (.deref))]
+          (is (= 200 (.getStatus response)))
+          (let [client-metrics (-> client
+                                   (.getMetricRegistry)
+                                   (Metrics/getClientMetrics))]
+            (is (.isEmpty (.getUrlTimers client-metrics)))
+            (is (.isEmpty (.getUrlAndMethodTimers client-metrics))))))
```

Comment:
And also it would be good to have a `testing` block saying what this test is testing.